source/playgrounds.txt

```diff
@@ -150,19 +150,41 @@ If you have a playground file open in |vscode-short| and do not have
 an active connection, |vsce| displays
 :guilabel:`Click here to add connection` at the top of your playground.
 
-.. figure:: /images/vsce-playground-add-connection-btn.png
-   :figwidth: 700px
-   :alt: Link to add connection from playground
-
-1. Click this link to open the connection string dialog.
-
-#. Enter the
-   :manual:`connection string </reference/connection-string/>` for the
-   deployment you want to run this playground against.
+.. procedure::
+   :style: normal
    
-#. Press :guilabel:`Enter`.
-
-#. Run your playground.
+   .. step:: Select :guilabel:`Click here to add connection` 
+
+      After you click :guilabel:`Click here to add connection`, |vsce| opens the 
+      connection string dialog.
+
+      .. figure:: /images/vsce-playground-add-connection-btn.png
+         :figwidth: 700px
+         :alt: Link to add connection from playground
+         
+   .. step:: Enter your connection string
```

Comment:
I had to click on extra time before the connection string dialog appeared:

<img width="615" alt="Screenshot 2023-12-08 at 5 34 38 PM" src="https://github.com/mongodb/docs-mongodb-vscode/assets/85948430/bde3d254-177d-443e-a436-bf27f0d214c6">

Reply:
Added an additional step about this + included information on how previously connected deployments will show up as options.